WcaOnRails/spec/controllers/delegate_reports_controller_spec.rb

```diff
@@ -88,6 +88,12 @@
       expect(comp.delegate_report.remarks).to eq "My newer remarks"
     end
 
+    it "updating report sets the discussion_url" do
```

Comment:
That sounds better to me. There won't be a discussion url until after it's posted anyways.